third_party/VENDOR-LICENSE/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2009 The Go Authors. All rights reserved.
```

Comment:
Did you run ./hack/update-codegen.sh

These seem like leftover that should be removed by that script.

Reply:
Yeah, robot is unhappy :)
https://github.com/knative-sandbox/eventing-rabbitmq/pull/317/checks?check_run_id=2692258987